package-lock.json

```diff
@@ -1,13 +1,13 @@
 {
     "name": "@moe-tech/orchestrator",
-    "version": "1.0.58",
+    "version": "1.0.61",
     "lockfileVersion": 1,
     "requires": true,
     "dependencies": {
         "2-thenable": {
             "version": "1.0.0",
-            "resolved": "https://registry.npmjs.org/2-thenable/-/2-thenable-1.0.0.tgz",
-            "integrity": "sha512-HqiDzaLDFCXkcCO/SwoyhRwqYtINFHF7t9BDRq4x90TOKNAJpiqUt9X5lQ08bwxYzc067HUywDjGySpebHcUpw==",
+            "resolved": "https://moetech.jfrog.io/artifactory/api/npm/npm/2-thenable/-/2-thenable-1.0.0.tgz",
```

Comment:
With open source projects, I don't think its a good practice to publish package-lock.json files with jfrog references.